src/alg_utils.jl

```diff
@@ -181,12 +182,6 @@ function DiffEqBase.prepare_alg(alg::Union{OrdinaryDiffEqAdaptiveImplicitAlgorit
       end
     end
 end
-function DiffEqBase.prepare_alg(alg::Union{OrdinaryDiffEqAdaptiveImplicitAlgorithm{0,AD,FDT},
```

Comment:
Accidental?

Reply:
I decided to change the approach to not set chunk size 1 for small enough duals.